third_party/rust/chromium_crates_io/patches/num-bigint-0.3.3/0101-num-bigint-handle-relative-OUT_DIR.patch

```diff
@@ -0,0 +1,25 @@
+From bd10c80b6daf0dc18d1f59874e634e502689d904 Mon Sep 17 00:00:00 2001
+From: Aleksey Khoroshilov <akhoroshilov@brave.com>
+Date: Tue, 29 Jul 2025 17:50:43 +0700
+Subject: [PATCH] num-bigint handle relative OUT_DIR.
+
+---
+ .../vendor/num-bigint-v0_3/src/biguint/convert.rs               | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/brave/third_party/rust/chromium_crates_io/vendor/num-bigint-v0_3/src/biguint/convert.rs b/brave/third_party/rust/chromium_crates_io/vendor/num-bigint-v0_3/src/biguint/convert.rs
+index c2e94792f45..c4ea13c12e4 100644
+--- a/brave/third_party/rust/chromium_crates_io/vendor/num-bigint-v0_3/src/biguint/convert.rs
++++ b/brave/third_party/rust/chromium_crates_io/vendor/num-bigint-v0_3/src/biguint/convert.rs
+@@ -723,7 +723,7 @@ pub(crate) fn to_str_radix_reversed(u: &BigUint, radix: u32) -> Vec<u8> {
+ #[inline]
+ fn get_radix_base(radix: u32, bits: u8) -> (BigDigit, usize) {
+     mod gen {
+-        include! { concat!(env!("OUT_DIR"), "/radix_bases.rs") }
++        include! { concat!("../", env!("OUT_DIR"), "/radix_bases.rs") }
+     }
+ 
+     debug_assert!(
+-- 
```

Comment:
looks like auto-formatting deleting some whitespace so please save without formatting and re-generate the patch

Reply:
I'll go ahead and approve assuming the small patch cleanup

Reply:
> looks like auto-formatting deleting some whitespace so please save without formatting and re-generate the patch


that's not a whitespace deletion, It's some kind of a patch separator, it exists in all `chromium_crates_ios` patches.